sql/core/src/main/scala/org/apache/spark/sql/catalog/Catalog.scala

```diff
@@ -477,6 +477,18 @@ abstract class Catalog {
   def cacheTable(tableName: String): Unit
 
   /**
+   * Caches the specified table in-memory with given storage level.
```

Comment:
`Caches the specified table in-memory with given storage level` -> 
`Caches the specified table with the given storage level`